build.sbt

```diff
@@ -67,7 +67,7 @@ lazy val commonScalacOptions = Seq(
   "-feature",
   "-unchecked",
   "-Xfatal-warnings",
-  "-Xlint",
+  "-Xlint:-missing-interpolator,_",
```

Comment:
I get a `possible missing interpolator: detected interpolated identifier '$conforms'` on https://github.com/adelbertc/frameless/pull/30/commits/943755747058060d636986e883f60f1eb971057e#diff-6917aeb0150d0a857fad88f6d2b7bb1aR67, I suspect that scalatest is doing something fishy...